components/Tutorial/content/bustHoaxes.js

```diff
@@ -175,50 +168,50 @@ export default [
     ],
   },
   {
-    title: '依照格式撰寫好讀回應',
-    subTitle: '查核完畢後，請依照建議格式寫下方便閱讀的查核結果',
-    content: `辛苦準備好回應材料之後，接下來就是寫下你查核闢謠的結果了！
-      為了讓你辛苦能被更多人看見，請依照建議格式寫下清晰好讀的查核結果，讓聊天機器人或是網站的使用者，能一眼看懂你的查核結果。`,
+    title: t`Write a reader-friendly response with our template`,
+    subTitle: t`After the checking process, please write down the result, presenting reader-friendly response following the suggested format.`,
+    content: t`The step is to compile and present the materials and findings that you have been put efforts in.
+    We advise writing clear responses in the suggested format to help the chatbot and Cofacts users easily read and understand your fact-checking results.`,
     subContent: [
       {
-        title: '選擇回應分類',
+        title: t`Select response category`,
         content: [
           {
             type: 'text',
-            data: `請根據你找到的資料，選擇「含有真實訊息」、「含有不實訊息」與「含有個人意見」。
-              若你找到的資料能說明訊息所說的部分不正確，可以將訊息標成「含有不實訊息」。
-              若你找到的資料大多也都是其他人的個人意見，只是看法與原訊息不同，那可以將訊息標成「含有個人意見」。
-              最後，如果你找到的資料說明訊息裡的資訊是正確的，那可以標成「含有正確訊息」。`,
+            data: t`Please assign tags of categories, "contains true information", "contains misinformation", or "contains personal perspective" to the content based on your findings.
+            
+            If your research result proves that the content includes false information, you can mark it as "contains misinformation".
+            
+            While your findings show personal perspectives divergent from the original, you can tag the content as "contains personal perspective".
+            
+            Finally, if the fact-checking result demonstrates that the content presents valid information, you can mark it as "contains true information".`,
           },
         ],
       },
       {
-        title: '填寫理由',
+        title: t`Fill in the reasons`,
         content: [
           {
             type: 'text',
-            data:
-              '選好了之後，簡短寫一下整段訊息裡，含有不實訊息/個人意見/真實訊息的部分在哪裡。這個部分的文字會顯示在 chatbot，建議使用「開門見山法」在第一段指出訊息哪裡錯誤、含有意見、或哪裡正確，若仍需更多說明，可以另行分段闡述。',
+            data: t`After tagging, write a brief note on which part in the content "contains true information", "contains misinformation", or "contains personal perspective". The note will be shown to users via our chatbot. We suggest clarifying false and true parts, or perspectives of the content in the first paragraph and elaborating in following paragraphs if more explanations are required.`,
           },
         ],
       },
       {
-        title: '放上找到的資料',
+        title: t`Provide sources and reference`,
         content: [
           {
             type: 'text',
-            data:
-              '在「參考資料」或「不同意見」欄位，貼上你找到的資料的連結們，就完成撰寫，可以送出囉！。',
+            data: t`Paste the links of information and list your reference in the "Reference" or "Other Opinions" column, then your response is good to go!`,
```

Comment:
"Other Opinions" has been renamed to "Opinion Sources"

![image](https://user-images.githubusercontent.com/108608/110197387-4bf50b80-7e86-11eb-8be9-ce5bd5675aa3.png)